src/renderer/components/publication/PublicationCard.tsx

```diff
@@ -65,13 +65,17 @@ class PublicationCard extends React.Component<PublicationCardProps, PublicationC
         const { publication, translator } = this.props;
         const authors = publication.authors.map((author) => translator.translateContentField(author)).join(", ");
         const MenuContent = this.props.menuContent;
+
         return (
             <div className={styles.block_book}
                 aria-haspopup="dialog"
                 aria-controls="dialog"
             >
                 <div className={styles.image_wrapper}>
-                    <a aria-hidden onClick={(e) => this.handleBookClick(e)}>
+                    <a
+                    tabIndex={0}
+                    onClick={(e) => this.handleBookClick(e)}
+                    onKeyPress={(e) => this.handleBookClick(e)}>
```

Comment:
We need to filter keyboard key-press events with `if (e.charCode === 13) { ... }` (`enter` key)